docs/site/Component.md

```diff
@@ -1,9 +1,10 @@
 ---
 lang: en
-title: 'Components'
+title: 'Component'
 keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Concepts
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Components.html
+permalink: /doc/en/lb4/Component.html
+redirect_from: /doc/en/lb4/Components.html
```

Comment:
+1 to use singular form in `permalink` and add `redirect_from` to preserve existing links 👍🏻